qiskit/tools/visualization/interactive/_iplot_state.py

```diff
@@ -18,14 +18,14 @@
 from ._iplot_qsphere import iplot_qsphere
 
 
-def iplot_state(quantum_state, method='city', options=None):
+def iplot_state(quantum_state, method='city', figsize=None):
```

Comment:
Should we also be raising a deprecation warning here? There are more kwargs for the individual method functions so it feels like we'll want to to do the same thing here that we did in the non-interactive version. Ditto for the input data check (maybe move `_validate_input_state` to `qiskit.tools.visualization._utils` and use it for the interactive functions too.